### PR TITLE
Fix typo in libcontainerd/client.go

### PR DESCRIPTION
**Pull request by xulike666**
**- What I did**
fix A typo and add .vscode/ in .gitignore

**- How I did it**
review code

**- How to verify it**
just  make code more readable :D 

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,6 +12,7 @@
 .editorconfig
 .gopath/
 .go-pkg-cache/
+.vscode/
```
Comment (AkihiroSuda) [inline .gitignore]: This should be in your ~/.gitignore_global (see https://github.com/docker/docker/pull/30060/files#diff-a084b794bc0759e7a6b77810e01874f2R2 )
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (xulike666) [inline .gitignore]: @AkihiroSuda thank you for this global .gitignore solution and that's much cooler.

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Review by cpuguy83 — APPROVED** [review]
Obviously failure is not from this change.

LGTM

**Pull request merged**